gcloud-java-storage/src/main/java/com/google/gcloud/spi/DefaultStorageRpc.java

```diff
@@ -521,4 +522,31 @@ public String open(StorageObject object, Map<Option, ?> options)
       throw translate(ex);
     }
   }
+
+  @Override
+  public RewriteResponse rewrite(StorageObject source, Map<Option, ?> sourceOptions,
+      StorageObject target, Map<Option, ?> targetOptions, String token, Long maxByteRewrittenPerCall)
+      throws StorageException {
+    try {
+      return storage
+          .objects()
+          .rewrite(source.getBucket(), source.getName(), target.getBucket(), target.getName(),
+              target)
+          .setRewriteToken(token)
+          .setMaxBytesRewrittenPerCall(maxByteRewrittenPerCall)
```

Comment:
Yes null sets no value.
And I agree we can make the API accept MB and do the multiplication in the RPC layer.